migrations/00278_contributions.sql

```diff
@@ -0,0 +1,622 @@
+
+COMMENT ON SEQUENCE "public"."contributions_id_seq"  IS NULL;
+
+--- END CREATE SEQUENCE "public"."contributions_id_seq" ---
+
+--- BEGIN ALTER TABLE "public"."mediaitems" ---
+
+ALTER TABLE IF EXISTS "public"."mediaitems" ADD COLUMN IF NOT EXISTS "primary_episode_id" int4 NULL  ;
+
+COMMENT ON COLUMN "public"."mediaitems"."primary_episode_id"  IS NULL;
+
+ALTER TABLE IF EXISTS "public"."mediaitems" ADD CONSTRAINT "mediaitems_primary_episode_id_foreign" FOREIGN KEY (primary_episode_id) REFERENCES episodes(id) ON DELETE SET NULL;
+
+COMMENT ON CONSTRAINT "mediaitems_primary_episode_id_foreign" ON "public"."mediaitems" IS NULL;
+
+--- END ALTER TABLE "public"."mediaitems" ---
+
+--- BEGIN CREATE TABLE "public"."contributions" ---
+
+CREATE TABLE IF NOT EXISTS "public"."contributions" (
+	"id" int4 NOT NULL DEFAULT nextval('contributions_id_seq'::regclass) ,
+	"user_created" uuid NULL  ,
+	"date_created" timestamptz NULL  ,
+	"user_updated" uuid NULL  ,
+	"date_updated" timestamptz NULL  ,
+	"person_id" uuid NOT NULL  ,
+	"type" varchar(255) NOT NULL DEFAULT NULL::character varying ,
+	CONSTRAINT "contributions_person_foreign" FOREIGN KEY (person_id) REFERENCES persons(id) ,
+	CONSTRAINT "contributions_pkey" PRIMARY KEY (id) ,
+	CONSTRAINT "contributions_user_created_foreign" FOREIGN KEY (user_created) REFERENCES directus_users(id) ,
+	CONSTRAINT "contributions_user_updated_foreign" FOREIGN KEY (user_updated) REFERENCES directus_users(id) 
+);
+
+ALTER TABLE IF EXISTS "public"."contributions" OWNER TO bccm;
+
+GRANT SELECT ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT INSERT ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT UPDATE ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT DELETE ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT TRUNCATE ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT REFERENCES ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT TRIGGER ON TABLE "public"."contributions" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
```

Comment:
`bccm` does not exist. The api user is called `api`
New things also need to allow `SELECT` by the backup user.